documentation/dashboard/working_with_data.md

```diff
@@ -268,18 +295,92 @@ for details.
+    by clicking the red 'remove' link on the right side of the forecast &
+    observation box.
 
      <img class="my-3" src="/images/report_form.png"/>
 
     After clicking submit, you will be returned to the report listing page where
     you will see the newly created report with a status of **pending**. The Arbiter
-    will process the report and then set its status to **complete**. You may then
-    view the web version of the report.
+    will process the report and then set its status to **complete** or
+    **failed**. You may then view the web version of the report. Failed reports
+    will contain an error message about the failure.
+
+
+
+Results
+-------
+
+The Solar Forecast Arbiter is foremost a forecast evaluation platform. This
+section explains how to interpret the results of those evaluations.
+
+### Data Validation Results
+
+The Solar Forecast Arbiter performs validation on all Observation data. The
+data validation toolkit applies quality flags to values in observation data.
+The Solar Forecast Arbiter includes these quality flags with observation data
+in the form of a bit mask. The table below shows quality flags and their
+bitmask.
+
+{: .table}
+|Quality Flag|Bitmask|Integer Value|Description|
+|------------|-------|-------------|-----------|
+|OK|0000000000000000|0|No validation flags.|
+|USER FLAGGED|0000000000000001|1|Providing user flagged data as problematic.|
+|NIGHTTIME|0000000000010000|16|Value occurs at night.|
+|CLEARSKY|0000000000010000|32|Values consistent with clear sky condition.|
+|SHADED|0000000001000000|64|Values indicate module shading.|
+|UNEVEN FREQUENCY|0000000010000000|128|Difference in timestamp between value and previous value does not conform to observation frequency.|
+|LIMITS EXCEEDED|0000000100000000|256|Physical limits exceeded. Uses QCrad criteria.|
+|CLEARSKY EXCEEDED|0000010000000000|512|Value greater than clearsky value.|
+|STALE VALUES|0000100000000000|1024||
+|INTERPOLATED VALUES|0001000000000000|2048|Values appear linear, suggesting interpolation.|
+|CLIPPED VALUES|0010000000000000|4096|Values indicate possible clipped power levels.|
+|INCONSISTENT IRRADIANCE COMPONENTS|0100000000000000|8192|GHI, DHI, and DNI are inconsistent.|
+|DAILY VALIDATION APPLIED|1000000000000000|16384||
+
+
+
+#### Quality Flags on the Dashboard
+
+On the Dashboard quality flags are plotted along with their timeseries values.
+Each type of quality flag has it's own colored bar running parallel to the x
+axis of the timeseries plots. A vertical colored bar indicates that the data
+was flagged at that timestamp while a solid colored blocks indicate the data
+was flagged within the colored range.
+
+<img class="my-3" src="/images/quality_flag_plot.png"/>
+
+
+### Report Analysis Results
+
+Solar Forecast Arbiter reports contain timeseries plots as well as a plot for
+ each metric and category selected during report creation
+(see [Create New Report](/dashboard/working-with-data/#create-new-report)).
```

Comment:
This doesn't do much for me. Maybe it would be worthwhile to add a report documentation section but I think they're self explanatory enough to leave for a later version.